src/app/shared/constants/options/spell_dbc_base_options.ts

```diff
@@ -115,3 +115,10 @@ export const SPELL_DBC_TARGET_AURA_STATE: Option[] = [
   { value: 6, name: 'Deadly Poison' },
   { value: 7, name: 'Bleeding' },
 ];
+
+export const SPELL_DBC_DAMAGE_CLASS: Option[] = [
+  { value: 0, name: 'NONE' },
+  { value: 0, name: 'MAGIC' },
+  { value: 0, name: 'MELEE' },
+  { value: 0, name: 'RANGED' },
```

Comment:
all the values are 0?

Reply:
Oops. Copy-pasted but didn't change the value. Fixed